src/util/logReadQueryError.js

```diff
@@ -1,10 +1,9 @@
-import Raven from 'raven-js';
+import * as Sentry from '@sentry/browser';
 
 export default (error, queryType = 'undefined_query_error_type') => {
 	console.error(queryType, error);
-	Raven.captureException(error, {
-		tags: {
-			query_error_type: queryType,
-		}
+	Sentry.withScope(scope => {
+		scope.setTag('query_error_type', queryType);
+		Sentry.captureException(error);
```

Comment:
@emuvente @eddieferrer 

I think we can drop the sentry call here and just leave the console.error which will continue to reveal the SSR invariant errors in our logs.

I'm going to take out the Raven call in the new release branch now.

Reply:
👍